babel.config.cjs

```diff
@@ -47,6 +47,7 @@ module.exports = function (api) {
     ],
     plugins: [
       '@wordpress/babel-plugin-import-jsx-pragma',
+      '@babel/transform-object-rest-spread',
```

Comment:
Why was this added?

Every browser supports object spread, no need to transform it.

Reply:
https://github.com/JetBrains/svg-sprite-loader/blob/c760cf93c49736f1e26a85fff7db3f9c940572c3/examples/custom-runtime-generator/webpack.config.js#L7C14-L7C42

It was there in the custom runtime generator example. But yes, I'll remove this.